types/inline-style-prefixer/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for inline-style-prefixer 3.0
+// Project: https://github.com/rofrischmann/inline-style-prefixer
+// Definitions by: Andrej Hazucha <https://github.com/ahz>
+//                 dpetrezselyova <https://github.com/dpetrezselyova>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default InlineStylePrefixer;
```

Comment:
It looks like this module directly exports the `Prefixer` class, and not a module with a `default` property. See the [README](https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).